### PR TITLE
Adds ResearchStudy aggregation Observations

### PR DESCRIPTION
**Pull request by bwalsh**
This PR contains the definitions and sample instances to create  `Observations` on a ResearchStudy containing aggregations [Consortium, StudyCohort, Summary, CohortCount, SampleCount, Participant, StorageSize]

Thanks @torstees

### CODE REVIEW

File: input/fsh/Alias.fsh [hunk 1]
```diff
@@ -51,3 +51,8 @@ Alias: $disease-codes = http://fhir.ncpi-project-forge.io/ValueSet/disease-codes
 Alias: $phenotype-codes = http://fhir.ncpi-project-forge.io/ValueSet/phenotype-codes
 Alias: $loinc-answer-codes = http://fhir.ncpi-project-forge.io/ValueSet/loinc-answers
 Alias: $condition-interitanc-codes = http://fhir.ncpi-project-forge.io/ValueSet/condition-inheritance-vs
+
+// NCPI Observations
+Alias: $ncpi = https://www.ncbi.nlm.nih.gov/fhir
```
Comment (torstees) [inline input/fsh/Alias.fsh:56]: I'm not sure this is actually the correct URL for ncpi. You may have gotten that from my examples, which were put together rather hastily. I only noticed I had grabbed nc-b-i after I had sent those files off.

**Review by torstees — COMMENTED** [review]
This looks good to me, except that ncbi url. I'm not sure what the right URL should be: 

https://ncpi-fhir.github.io is the prefix that our profiles currently have, as far as I'm aware, however, I think that is out of date after the somewhat recent change in the NCPI group's name. So, maybe it should be _nih-ncpi.github.io/fhir_? Regardless, that doesn't currently exist, but seems like one we might be able to control via github unless I'm misunderstanding how the github.io subdomains work.

**Review by torstees — APPROVED** [review]
Looks fine except for the NCPI alias, which I think needs to be something else. Maybe not important right now, though (see comments to PR)

**Review by torstees — APPROVED** [review]
looks good

**Comment by liberaliscomputing** [comment]
@RobertJCarroll, @allisonheath, and @torstees, I want to open up a discussion about this PR since:

- I am not sure whether using Observation for research metadata is within right scope and boundaries; and
- If I understand correctly, `component` should be part of `value[x]` while this approach doesn't populate `value[x]`

I am down for the idea of not creating extensions when not necessary, but the above are my concerns.

**Comment by RobertJCarroll** [comment]
Can you point to the documentation that indicates it is outside of scope? Based on the inclusion of "group" as an acceptable subject reference type, it seems reasonable. Subject: "The patient, or group of patients, location, or device this observation is about and into whose record the observation is placed. "

Can you show us the documentation that describes this limitation? Components are separate from the value of the core observation. The only thing I can find connecting them is Obs-7 that says: "If Observation.code is the same as an Observation.component.code then the value element associated with the code SHALL NOT be present", which doesn't apply as far as I can tell as our components are not using the same code as the observation.

**Comment by bwalsh** [comment]
One piece additional evidence supporting the idea of using observations to report on ResearchStudies.
This term exists to define terms that apply to a "cohort"

From https://terminology.hl7.org/2.1.0/CodeSystem-v3-ObservationValue.html

image

**Comment by bwalsh** [comment]
@liberaliscomputing @RobertJCarroll @torstees :  thoughts?  can we merge?

**Review by liberaliscomputing — APPROVED** [review]

**Pull request merged**